src/main/resources/l10n/JabRef_en.properties

```
@@ -2580,6 +2580,8 @@ Finished\ writing\ metadata\ for\ library\ %0\ (%1\ succeeded,\ %2\ skipped,\ %3
 Processing...=Processing...
 Writing\ metadata\ to\ %0=Writing metadata to %0
 
+Get\ More\ Themes= Get More Themes
```

Had to remove the space after `=`

```suggestion
Get\ more\ themes...=Get more themes...
```